tplinkrouterc6u/client/c6u.py

```diff
@@ -360,6 +360,9 @@ def _add_device(conn: Connection, item: dict) -> None:
                 devices[item['mac']].up_speed = item.get('uploadSpeed')
                 devices[item['mac']].signal = int(item.get('signal')) if item.get('signal') else None
 
+        # Store main wifi count (so only the total will be incremented for any new devices from wireless statistics array)
+        status.main_wifi_clients_total = status.wifi_clients_total
```

Comment:
It is better not to add new counters, because there are also other clients that dont have it.